src/ruby/ext/grpc/extconf.rb

```diff
@@ -70,7 +71,8 @@
   puts 'Building internal gRPC into ' + grpc_lib_dir
   nproc = 4
   nproc = Etc.nprocessors * 2 if Etc.respond_to? :nprocessors
-  system("make -j#{nproc} -C #{grpc_root} #{grpc_lib_dir}/libgrpc.a CONFIG=#{grpc_config} Q=")
```

Comment:
Probably needs to happen for FreeBSD too?

Reply:
I'm not sure, I've never used FreeBSD.

Reply:
My point is, I think the platform-specific code in the Ruby build system should be unified for all BSDs, not just special case OpenBSD.

Reply:
If I recall correctly, you had to use `gmake` on FreeBSD as well. Let's run `gmake` for all BSDs.

Reply:
NetBSD/FreeBSD require gmake for GNU features.

Reply:
I've made this so that all the BSD's will use `gmake` in a new commit.